hack/README.md

```diff
@@ -0,0 +1,69 @@
+docker/hack
+============================
+
+## About
+
+`hack` contains a collection of scripts used by Docker. If there are any issues regarding the intention of a particular script
```

Comment:
@abhishek7 let's use `Docker` for now, when `moby` is going to get split out, these files might change a lot so.. let's not change that for now 😉